src/acl/AtStep.cc

```diff
@@ -8,25 +8,29 @@
 
 #include "squid.h"
 
-#if USE_OPENSSL
-
 #include "acl/AtStep.h"
 #include "acl/AtStepData.h"
 #include "acl/FilledChecklist.h"
 #include "client_side.h"
 #include "http/Stream.h"
+#if USE_OPENSSL
 #include "ssl/ServerBump.h"
+#endif
 
 int
-ACLAtStepStrategy::match (ACLData<Ssl::BumpStep> * &data, ACLFilledChecklist *checklist)
+ACLAtStepStrategy::match (ACLData<XactionSteps> * &data, ACLFilledChecklist *checklist)
 {
+    Must(checklist->request);
+    Must(checklist->request->masterXaction);
+    if (checklist->request->masterXaction->generatingConnect && data->match(xaStepGeneratingConnect))
```

Comment:
Can we move this check _after_ SslBump checks (because most existing uses will be for SslBump step values)?

Reply:
yep.
Fixed with commit 8f212b54d6